### PR TITLE
Added testcase to verify fix (MLDB-1503)

### PR DESCRIPTION
**Pull request by jeremybarnes**
Simple testcase to verify fix.

### CODE REVIEW

File: testing/MLDB-1503-bad-rowname-error-message.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+function assertEqual(expr, val, msg)
+{
+    if (expr == val)
+        return;
+    if (JSON.stringify(expr) == JSON.stringify(val))
+        return;
+
+    throw "Assertion failure: " + msg + ": " + JSON.stringify(expr)
+        + " not equal to " + JSON.stringify(val);
+}
```
Comment (FinchPowers) [inline testing/MLDB-1503-bad-rowname-error-message.js]: Unrelated: Any way we can extract that? We could consider putting it into mldb js handle. Copy/pasting it everywhere clutters every test IMO.

**Comment by FinchPowers** [comment]
I assume that before the fix that test was failing? +1.

File: testing/MLDB-1503-bad-rowname-error-message.js (hunk 1 above)
Comment (jeremybarnes) [inline testing/MLDB-1503-bad-rowname-error-message.js]: Yes, I would like to add the ability to require a js file with this kind of thing in.  We should add it one day soon...

**Comment by jeremybarnes** [comment]
No, it wasn't failing before.  At some point of time in the past it was but was fixed in the meantime; this takes the reproducing case from the ticket and makes sure it stays fixed.

**Pull request merged**